### ISSUE TITLE
Failing to deploy

### ISSUE THREAD

**Issue by toby-freemarket** [OctopusDeploy/deploy-release-tenanted-action#13]
**Describe the bug**
Hi I'm not sure what I'm doing wrong.  I seem to be able to create a release but not deploy it.
The error is 
`Error: There was a problem with your request.
Exception has been thrown by the target of an invocation.`
I've included what we have in our yaml  and the relating logs, There is nothing in the octopus audit either

Any ideas on how to debug this?

**To Reproduce**
```yaml
      - name: 🐙 Deploy a release
        uses: OctopusDeploy/deploy-release-tenanted-action@v3
        with:
          project: 'TeamAmerica'
          release_number: ${{ env.VERSION_NUMBER }}
          environment: 'Dev'
          tenants: |
            'UK'
          tenant_tags: |
            'UK'
```
**Logs**
```logs
2023-01-29T22:28:26.1557456Z 🐙 Creating a release in Octopus Deploy...
2023-01-29T22:28:29.2055443Z 🎉 Release 1.0.196 created successfully!
2023-01-29T22:28:29.2465374Z ##[group]Run OctopusDeploy/deploy-release-tenanted-action@v3
2023-01-29T22:28:29.2465798Z with:
2023-01-29T22:28:29.2466012Z   project: TeamAmerica
2023-01-29T22:28:29.2466251Z   release_number: 1.0.196
2023-01-29T22:28:29.2466487Z   environment: Dev
2023-01-29T22:28:29.2466711Z   tenants: 'UK'

2023-01-29T22:28:29.2466918Z   tenant_tags: 'UK'

2023-01-29T22:28:29.2467157Z   use_guided_failure: false
2023-01-29T22:28:29.2467375Z env:
2023-01-29T22:28:29.2467739Z   OCTOPUS_URL: ***
2023-01-29T22:28:29.2468069Z   OCTOPUS_API_KEY: ***
2023-01-29T22:28:29.2468305Z   OCTOPUS_SPACE: Default
2023-01-29T22:28:29.2468542Z   VERSION_NUMBER: 1.0.196
2023-01-29T22:28:29.2468755Z ##[endgroup]
2023-01-29T22:28:30.1263749Z 🐙 Deploying a release in Octopus Deploy...
2023-01-29T22:28:31.3446737Z ##[error]Error: There was a problem with your request.
Exception has been thrown by the target of an invocation.
2023-01-29T22:28:31.3703521Z Post job cleanup.
```

**Expected behavior**
Deploy to the UK tenant

**Comment by benPearce1** [comment]
@toby-freemarket there are a couple of expected reasons why this may fail:

- The release cannot be deployed for a specified tenant because they are not connected to the project or the user with specified API key doesn't have permission.
- The tenant and/or tags specified do not match any tenants.

Are you values correct for the `UK` tenant or should it just be a tag?
Also, the tag should include the tag set name, in the format `tag set/tag name`.

**Comment by benPearce1** [comment]
[sc-42529]

**Comment by benPearce1** [comment]
Unable to replicate, no further feedback from customer

**Comment by toby-freemarket** [comment]
Sorry I completely missing this,  let me try again and come back to you
